tools/backup_analysis.py

```diff
@@ -257,12 +261,13 @@ def compare_file(olderdir, newerdir, opts, filename, cmp_size):
     if not opts.terse:
         # Print the time even if no changes because we may want to know how long it took to not
```

Comment:
It looks like this comment is out of date. If the file has no changes, we return without printing the time.

Reply:
Fixed.